orchagent/portsorch.cpp

```diff
@@ -8104,6 +8128,27 @@ void PortsOrch::getPortSerdesVal(const std::string& val_str,
     }
 }
 
+ReturnCode PortsOrch::setPortLoopbackMode(sai_object_id_t id, sai_port_loopback_mode_t loopback_mode)
+{
+    SWSS_LOG_ENTER();
+
+    sai_attribute_t attr;
+    attr.id = SAI_PORT_ATTR_LOOPBACK_MODE;
+    attr.value.u32 = loopback_mode;
+
+    sai_status_t status = sai_port_api->set_port_attribute(id, &attr);
+    if (status != SAI_STATUS_SUCCESS)
+    {
+        LOG_ERROR_AND_RETURN(ReturnCode(status)
```

Comment:
@royyi8 this won't crash OA right?

Reply:
@royyi8 can we avoid crash. Why crash OA if one port fails?

Reply:
@prgeor To clarify, the code will not crash OA. It will simply log an error and return status failed in the `setPortLoopbackMode` function. OA will proceed in the else block on line 4319.